tests/test.ts

```diff
@@ -24,25 +24,39 @@ for (const fileName of fs.readdirSync('cases')) {
     describe("Generating baseline for " + fileName, () => {
         const text = fs.readFileSync(path.join('./cases', fileName), 'utf8');
         const parsedFileName = path.parse(fileName);
-        const { markerScopes, wholeBaseline } = build.generateScopes(text, parsedFileName);
-        if (markerScopes) {
-            addTestCase(parsedFileName.name + '.txt', markerScopes);
-        }
-        addTestCase(parsedFileName.name + '.baseline.txt', wholeBaseline);
+
+        const generateScopes = (() => {
+            let result: { markerScopes: string, wholeBaseline: string };
```

Comment:
Nit.. May be add this to before() function in describe and hoist the markerScopes and wholeBaseline up.. then this function expression inside another one isn't needed.